android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java

```diff
@@ -68,25 +71,36 @@ public void onRefresh() {
         sponsorsRecyclerView.addOnItemTouchListener(new RecyclerItemClickListener(view.getContext(), new RecyclerItemClickListener.OnItemClickListener() {
             @Override
             public void onItemClick(View itemView, int position) {
-                String sponsorUrl = dbSingleton.getSponsorList().get(position).getUrl();
-                if (!sponsorUrl.startsWith("http") || !sponsorUrl.startsWith("https")) {
+                List<Sponsor> objects = dbSingleton.getSponsorList();
+                Sponsor sponsor = objects.get(position);
+                String sponsorUrl = sponsor.getUrl();
+                if (!sponsorUrl.startsWith("http") && !sponsorUrl.startsWith("https")) {
```

Comment:
No @the-dagger. I have fixed it here. Earlier it was check if it doesn't start with http/https, it would append but obviously the url can't start with both. It starts with one of them, so the and.